client/src/pages/GameBoard.js

```diff
@@ -0,0 +1,42 @@
+/*
+UI for Game Board
+*/
+
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid, Paper } from '@material-ui/core';
+
+const useStyles = makeStyles({
+    root: {
+      flexGrow: 1,
+      marginTop: '20px',
+      display: 'grid',
+      gridTemplateColumns: 'auto auto auto auto auto'
+    },
```

Comment:
It's not great practice to redefine the grid properties of Material UI but in this case I'm not sure there's a not-hacky way to get five columns. If Jess and your team are okay with it then I think this is fine

Reply:
I think it would make sense to use Box components instead of combining both Flexbox and CSS Grid

Reply:
I have changed to the box component and implemented onClick functionality on each card then the color changes to grey.

<img width="1440" alt="Screen Shot 2021-04-21 at 11 48 51 AM" src="https://user-images.githubusercontent.com/46986531/115583086-8fb99e00-a297-11eb-9fcf-4d332f6ae8ce.png">